<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>blocNotes</class>
 <widget class="QMainWindow" name="blocNotes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bloc-notes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressourcesicones.qrc">
    <normaloff>:/menu/icones/notes-icon.png</normaloff>:/menu/icones/notes-icon.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>85</red>
            <green>170</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>170</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>85</red>
            <green>170</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>170</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Midlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>85</red>
            <green>170</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>51</red>
            <green>153</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="cursorWidth">
       <number>2</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionEnregistrer_sous"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionPolice"/>
    <addaction name="actionCouleur_du_fond"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionAnnuler"/>
    <addaction name="actionR_tablir"/>
    <addaction name="separator"/>
    <addaction name="actionRechercher"/>
    <addaction name="actionRechercher_le_suivant"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuFormat"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouveau"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionEnregistrer"/>
   <addaction name="separator"/>
   <addaction name="actionAnnuler"/>
   <addaction name="actionR_tablir"/>
   <addaction name="actionRechercher"/>
   <addaction name="separator"/>
   <addaction name="actionA_propos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouveau">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/page-blanche-icon.png</normaloff>:/menu/icones/page-blanche-icon.png</iconset>
   </property>
   <property name="text">
    <string>Nouveau</string>
   </property>
   <property name="toolTip">
    <string>Nouveau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/open-file-icon.png</normaloff>:/menu/icones/open-file-icon.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/Save-icon.png</normaloff>:/menu/icones/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/Save-icon.png</normaloff>:/menu/icones/Save-icon.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer sous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPolice">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/font-icon.png</normaloff>:/menu/icones/font-icon.png</iconset>
   </property>
   <property name="text">
    <string>Police</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/point-interrogation-icon.png</normaloff>:/menu/icones/point-interrogation-icon.png</iconset>
   </property>
   <property name="text">
    <string>A propos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/Windows-Close-Program-icon.png</normaloff>:/menu/icones/Windows-Close-Program-icon.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionCouleur_du_fond">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/color-icon.png</normaloff>:/menu/icones/color-icon.png</iconset>
   </property>
   <property name="text">
    <string>Couleur du fond</string>
   </property>
  </action>
  <action name="actionAnnuler">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/defaire-icon.png</normaloff>:/menu/icones/defaire-icon.png</iconset>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionR_tablir">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/refaire-icon.png</normaloff>:/menu/icones/refaire-icon.png</iconset>
   </property>
   <property name="text">
    <string>RÃ©tablir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionRechercher">
   <property name="icon">
    <iconset resource="ressourcesicones.qrc">
     <normaloff>:/menu/icones/recherche-icon.png</normaloff>:/menu/icones/recherche-icon.png</iconset>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionRechercher_le_suivant">
   <property name="text">
    <string>Rechercher le suivant</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressourcesicones.qrc"/>
 </resources>
 <connections/>
</ui>
